src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -552,7 +551,7 @@ export default function Swap({
               <SwitchToWethBtn wrappedToken={wrappedToken} />
             ) : isFeeGreater ? (
               <FeesExceedFromAmountMessage />
-            ) : isInsufficientLiquidityError(quote?.error) ? (
+            ) : quote?.error === 'insufficient-liquidity' ? (
```

Comment:
Handling our error is easier now. Since the error will contain relevant errors we handle, is easy to map it to UI behaviour